tensorflow_lite_support/java/src/java/org/tensorflow/lite/support/image/BoundingBoxUtil.java

```diff
@@ -197,6 +197,42 @@ private static RectF convertFromBoundaries(
     }
   }
 
+  private static RectF convertFromUpperLeft(
+      float[] values, CoordinateType coordinateType, int height, int width) {
+    if (coordinateType == CoordinateType.PIXEL) {
+      float left = values[0];
+      float top = values[1];
+      float w = values[2];
+      float h = values[3];
+
+      return new RectF(
+        left, top, left+w, top+h);
+    } else {
+      return new RectF(
+        values[0] * width, values[1] * height, (float) width, (float) height);
+    }
+  }
+
+  private static RectF convertFromCenter(
+      float[] values, CoordinateType coordinateType, int height, int width) {
+    if (coordinateType == CoordinateType.PIXEL) {
+      float centerX = values[0];
+      float centerY = values[1];
+      float w = values[2];
+      float h = values[3];
+
+      float left = centerX - w/2;
+      float top = centerY - h/2;
+      float right = centerX + w/2;
+      float bottom = centerX + h/2;
+
+      return new RectF(
+        left, top, right, bottom);
+    } else {
+      throw new IllegalArgumentException("BoundingBox.Type Center is not supported with CoordinateType CoordinateType.RATIO");
```

Comment:
IMHO, it looks like able to be supported - perhaps leave a TODO here if you don't want to put all in one PR.

Reply:
@xunkai55 Thanks for reviewing, I have made other changes, please check.

Reply:
values[2] and [3] represent the relative width and height of the bounding box. So the height and width of the box should be `values[2] * width`, or `imageWidth` as I commented in another thread.